src/App.js

```diff
@@ -1,29 +1,19 @@
 import React from 'react';
-
-import './App.css';
-
 import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
+import PostList from './Components/PostList';
+
+const preparedPostsWithUsersAndComments = posts.map(post => ({
+  ...post,
+  user: users.find(user => user.id === post.userId),
+  comment: comments.filter(comment => comment.postId === post.id),
+}));
 
 const App = () => (
   <div className="App">
     <h1>Static list of posts</h1>
-
-    <p>
-      <span>posts: </span>
-      {posts.length}
-    </p>
-
-    <p>
-      <span>comments: </span>
-      {comments.length}
-    </p>
-
-    <p>
-      <span>Users: </span>
-      {users.length}
-    </p>
+    <PostList posts={preparedPostsWithUsersAndComments} />
```

Comment:
```suggestion
    <PostList posts={combinedPostData} />
```
Не делай такие длинные названия переменных

Reply:
+